whatlies/embeddingset.py

```diff
@@ -892,8 +892,11 @@ def plot_movement(
 
     def plot_interactive(
         self,
-        x_axis: Union[str, Embedding],
-        y_axis: Union[str, Embedding],
+        x_axis: Union[int, str, Embedding],
+        y_axis: Union[int, str, Embedding],
```

Comment:
Don't we want to add a default value for these? Unless specified I can imagine a use-case to have `x_axis`=0 and `y_axis=1`. That way you don't have to fiddle around when you're using `Umap` or `Pca`.

Reply:
@koaning Actually, I had that in mind and wanted to create an issue for it since this is also a valid point for `plot` method as well (and even `Embedding.plot`). If you are OK with it, I'll address this in a separate PR with `x_axis=0` and `y_axis=1` as the default for all plot methods. Is that fine with you?

Reply:
Ah yeah, that's a better idea. Makes the testing a lot clearer too.